materials/exports/vars.js

```diff
@@ -0,0 +1,113 @@
+module.exports = {
+  customProperties: {
```

Comment:
Just curious: is this something we can set? Why is it "wrapped"?

Reply:
@tdeekens this is the shape that [postcss-custom-properties](https://github.com/postcss/postcss-custom-properties) generates and expects for it's importFrom. 

There's an example [here](https://github.com/postcss/postcss-custom-properties/blob/master/test/import-properties.js) as well

Reply:
Thanks.